packages/sources/react/src/components/navigation/VtmnNavbarLink/VtmnNavbarLink.tsx

```diff
@@ -34,35 +24,24 @@ export interface VtmnNavbarLinkProps
 
 export const VtmnNavbarLink = ({
   icon,
-  label,
-  showLabel = true,
   className,
   children,
   ...props
 }: VtmnNavbarLinkProps) => {
   const relAttrValues = props?.target && computeRel(props.target, props.rel);
-  return showLabel ? (
+  const childrenArray = React.Children.toArray(children);
+  return (
     <a
       className={clsx('vtmn-navbar_link', className)}
       {...props}
       {...(relAttrValues && { rel: relAttrValues })}
     >
-      {children}
-      <VtmnIcon value={icon} aria-hidden={true} />
-      {label}
-    </a>
-  ) : (
-    <a
-      className={clsx(
-        'vtmn-navbar_link',
-        'vtmn-navbar_link--icon-alone',
-        className,
-      )}
-      {...props}
-    >
-      {children}
+      {childrenArray.find((child: ReactElement) => {
+        console.log(child);
+        return child.type === VtmnBadge;
+      })}
       <VtmnIcon value={icon} aria-hidden={true} />
-      <span className="vtmn-sr-only">{label}</span>
+      {childrenArray.find((child: ReactElement) => child.type === 'span')}
```

Comment:
Better be sure that this condition can always be fulfilled... Should we add a fallback here ?

Reply:
ping @Tlahey :)

Reply:
On other framework no fallback options are defined.
We should explained on the doc that the span are required with label.

Reply:
I check if undefined return an error, if not, I return an empty string

Reply:
![image](https://github.com/Decathlon/vitamin-web/assets/2856778/d7c8aac4-cc3f-47fa-84eb-f2acdc9c47e1)

With react, without span it's good